### PR TITLE
fix modal

### PR DESCRIPTION
**Pull request by dimasster**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by EyR1oN — APPROVED** [review]

File: src/features/StreetcodePage/RelatedFiguresBlock/RelatedFigureItem/RelatedFigureItem.component.tsx [hunk 1]
```diff
@@ -38,9 +38,9 @@ const RelatedFigureItem = ({ relatedFigure, filterTags = false }: Props) => {
                 </p>
                 <div className="relatedTagList">
                     {tags.filter((tag) => getTagArray.find((ti) => ti.id === tag.id || !filterTags))
-                        .map((tag, idx) => (
+                        .map((tag) => (
                         // eslint-disable-next-line react/no-array-index-key
-                            <div key={idx} className="tag">
+                            <div key={tag.id} className="tag">
                                 <p>{tag.title}</p>
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/RelatedFiguresBlock/RelatedFigureItem/RelatedFigureItem.component.tsx:44]: Consider using argument destructuring here.

File: src/app/common/components/modals/RelatedFigures/RelatedFiguresModal.styles.scss [hunk 2]
```diff
@@ -52,14 +52,22 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+        @include mut.sized($width: 272px, $height: 97px);
+    }
+  }
+
+  .relatedFigureSlide:hover {
+    @include mut.sized($width: 278px, $height: 380px);
 
     .slideText{
-        @include mut.sized($width: 272px, $height: 169px);
+        @include mut.sized($width: 272px, $height: 97px);
```
Comment (AleXLaeR) [inline src/app/common/components/modals/RelatedFigures/RelatedFiguresModal.styles.scss:67]: It is not necessary to type out mixin parameter names. But if you're doing this for the declarative nature, it is perfectly fine.

Comment (AleXLaeR) [inline src/app/common/components/modals/RelatedFigures/RelatedFiguresModal.styles.scss:67]: This selectors may be incapsulated in the block above (l. 52).

File: src/features/StreetcodePage/RelatedFiguresBlock/RelatedFigureItem/RelatedFigureItem.styles.scss [hunk 3]
```diff
@@ -31,7 +31,8 @@
 
         .heading {
             @include mut.sized($width: 100%, $height: 88px);
-            @include mut.flexed($align-items: center);
+
+            @include mut.flex-centered();
             @include mut.rem-padded($left: 12px, $right: 12px, $top: 10px);
 
             @include mut.with-font($font-family: f.$closer-text-font, $font-size: 20px, $font-weight: 500);
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/RelatedFiguresBlock/RelatedFigureItem/RelatedFigureItem.styles.scss:38]: font-size is 20px by default.

**Review by AleXLaeR — APPROVED** [review]

**Review by Tatiana2424 — APPROVED** [review]

**Pull request merged**